sonar-db/src/main/java/org/sonar/db/DefaultDatabase.java

```diff
@@ -168,6 +180,6 @@ private static void completeDefaultProperty(Properties props, String key, String
 
   @Override
   public String toString() {
-    return "Database[" + (properties != null ? properties.getProperty(SONAR_JDBC_URL) : "?") + "]";
+    return format("Database[%s]", properties != null ? properties.getProperty(SONAR_JDBC_URL) : "?");
```

Comment:
Not sure using String.format in toString() is generally a good practice.
toString() can be highly performance sensitive and using a simple string concatenation is better than using format to that regard